torch/csrc/jit/ir.h

```diff
@@ -63,6 +63,9 @@ using ::c10::getTypePtr;
 using ::c10::MatchTypeReturn;
 using ::c10::TypeKind;
 
+using ::c10::filter;
+using ::c10::fmap;
```

Comment:
This seems weird to me. The other user declarations make sense because they are reexporting "IR" stuff in the torch namespace. But then why would some random functional combinators also come along for the ride?

However, I am guessing you added this, because you would have had to renamespace a bunch of other stuff if you didn't? Maybe this suggests that we should have a global `using namespace c10` in torch...

Reply:
Originally I put it here mostly to keep the changes mechanical (that's what we included with `jit/functional.h` anyway).

Now I removed `using c10::filter` from ir.h as it was used in only one place, but kept `c10::fmap`. Hopefully, it will keep annoying us so that we don't forget to cleanup this and unify namespaces.